Competition/src/main/java/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -131,6 +135,14 @@ public void setupElevatorCommands(OperatorInterface operatorInterface, RaiseElev
 
         operatorInterface.operatorGamepad.getifAvailable(8).whenPressed(stopElevator);
         operatorInterface.operatorGamepad.getifAvailable(9).toggleWhenPressed(override);
+
+        setElevatorLow.setGoal(elevatorSubsystem.getTickHeightForLevel(HatchLevel.Low));
+        setElevatorMid.setGoal(elevatorSubsystem.getTickHeightForLevel(HatchLevel.Medium));
+        setElevatorHigh.setGoal(elevatorSubsystem.getTickHeightForLevel(HatchLevel.High));
+
+        operatorInterface.operatorGamepad.getPovIfAvailable(0).whenPressed(setElevatorLow);
```

Comment:
0 == Up, 90 == Right, 180 == down, 270 == left. Is that what you were looking for?